IDR/notebook/scripts/nbconverted/1.describe_screens.py

```diff
@@ -267,51 +390,46 @@ def describe_screen(screen_id):
 screen_details_df.to_csv(output_file, index=False, sep="\t")
 
 
-# In[13]:
+# In the chunk below, we collect and process a subset of the available IDR screens. The subset contains IDR accession names idr0080, idr0001, and idr0069. The idr0080 dataset was collected by Way et al. (2021) and will be used as a validation set for our statistics pipeline as we already know the image counts and unique attribute counts. The other two datasets are used for prototyping iterative functionalities of metadata collection and statistics.
+
+# In[40]:
+
+
+# Collect study_names and imaging method metadata for each screen
+idr_names_dict = dict()
+for index in screen_details_df.itertuples(index=False):
+    screenID = index[19]
+    idr_name = index[18]
+    img_type = index[5]
+    sample = index[0]
```

Comment:
these indexes scare me 😱 - is hardcoding the index the only option here? I feel like this hardcoding is fragile, but maybe I'm overreacting

Reply:
This was something I forgot to mention during our meeting. This was another placeholder and I just kept moving forward with it because it works for now. The index values are specific to how I set up the data frame so it "shouldn't" change. I will look into how to generalize it.

Reply:
sounds good. If it ends up being a rabbit hole, we can solve for generalizbility in production (or continue to test in prototyping in a future PR)

Reply:
@gwaybio I changed it so it now searches for the column names like `screen_id` or `idr_name` for example. This way, the column locations can be changed and those values will still be identified provided the names for those categories remain the same. The code I have now for this is repetitive, but haven't figured out how to iterate it yet. I don't want to make it more complicated than it needs to be.

Added in 7994ac7f55b94a793ae5a2243210e5f92f7e5148.